20230726/s7-1690407403663/README.md

```diff
@@ -0,0 +1,60 @@
+# Scenario 1 - ALS-bypass Baseline with Sims-only
```

Comment:
Is this the right description for this scenario? This is for Scenario 1

Reply:
Nice catch.

This should be `ALS Baseline with Sims-only, UV_THREADS`, I have fixed this and also renamed it to match you previous suggestions, i.e. `Sims-only` -> `Sims` = `ALS Baseline with Sims, UV_THREADS`.